l10n_it_fatturapa_in/models/company.py

```diff
@@ -43,7 +79,22 @@ def onchange_company_id(self):
                 company.sconto_maggiorazione_product_id and
                 company.sconto_maggiorazione_product_id.id or False
                 )
+            self.arrotondamenti_attivi_account_id = (
```

Comment:
Non serve onchange per i campi related

Reply:
Ok! L'onchange andrebbe rimosso del tutto dato che sono tutti related

Reply:
Sì

Reply:
Magari il resto lo togliamo in un commit separato oppure un'altra PR apposita, che dici?

Reply:
@robyf70 va bene commit separato, non credo valga la pena fare PR apposita